app/available-area/action-compatibility-matrix.js

```diff
@@ -4,12 +4,13 @@ const ACTION_COMPATIBILITY_MATRIX = {
   z: ['y'],
   za: ['zb'],
   zb: ['za'],
-  SAM1: ['SAM3', 'SAM2', 'AB3'],
-  SAM2: ['SAM1', 'SAM3'],
-  SAM3: ['SAM1', 'SAM2'],
-  LIG1: [],
-  AB3: ['SAM1'],
-  GRH1: ['SAM1']
+  SAM1: ['SAM3', 'SAM2', 'AB3', 'LIG1'], // correction LIG1 added
+  SAM2: ['SAM1', 'SAM3', 'CSAM1'],
+  SAM3: ['SAM1', 'SAM2', 'CSAM1'],
+  LIG1: ['SAM1'], // correction LIG1 added
+  AB3: ['SAM1', 'CSAM1'],
+  GRH1: ['SAM1'],
```

Comment:
Why would SAM1 be compatible but CSAM1 wouldn't? I would expect the C version of these wherever the older one is